administrator/language/de-DE/com_config.ini

```
@@ -60,19 +60,27 @@ COM_CONFIG_FIELD_DATABASE_NAME_LABEL="Datenbank"
 COM_CONFIG_FIELD_DATABASE_PASSWORD_LABEL="Passwort"
 COM_CONFIG_FIELD_DATABASE_PREFIX_LABEL="Präfix"
+COM_CONFIG_FIELD_DATABASE_TYPE_DESC="Der zu nutzende Datenbanktyp, der während der Installation eingegeben wurde.<br /><br /><strong>Warnung:</strong><br />Dieses Feld sollte nicht verändert werden, außer in speziellen Fällen, wie z.&#8239;B. beim Umzug der Datenbank zu einem neuen Webhosting-Provider."
 COM_CONFIG_FIELD_DATABASE_TYPE_LABEL="Typ"
 COM_CONFIG_FIELD_DATABASE_USERNAME_LABEL="Benutzer"
 COM_CONFIG_FIELD_DEBUG_CONST="Konstante"
 COM_CONFIG_FIELD_DEBUG_CONST_LANG_DESC="Auswählen, ob die Sprachkonstante oder der Sprachwert angezeigt werden sollen, wenn die Sprache debuggt wird."
 COM_CONFIG_FIELD_DEBUG_CONST_LANG_LABEL="Sprachanzeige"
+COM_CONFIG_FIELD_DEBUG_LANG_DESC="Auswählen, ob die Fehleranzeiger (<strong>** ... **</strong>) oder (<strong>?? ... ??</strong>) für die Joomla-Sprache angezeigt werden sollen. „Sprache debuggen“ funktioniert auch ohne die Aktivierung von „System debuggen“, zeigt dann allerdings nicht die benötigten zusätzlichen Informationen an, die bei der Korrektur der Fehler hilfreich wären."
 COM_CONFIG_FIELD_DEBUG_LANG_LABEL="Sprache debuggen"
+COM_CONFIG_FIELD_DEBUG_SYSTEM_DESC="Wenn aktiviert, werden diagnostische Informationen, Übersetzungen und SQL-Fehler (wenn vorhanden) angezeigt. Die Informationen werden auf jeder Seite im Joomla!-Backend und Frontend angezeigt.<br /><br /><strong>Warnung:</strong><br />Es wird ausdrücklich davon abgeraten, diese Informationen auf einer Liveseite anzuzeigen."
 COM_CONFIG_FIELD_DEBUG_SYSTEM_LABEL="System debuggen"
 COM_CONFIG_FIELD_DEBUG_VALUE="Wert"
+COM_CONFIG_FIELD_DEFAULT_ACCESS_LEVEL_DESC="Die Standard-Zugriffsebene für neue Inhalte, Menüeinträge und andere Einträge."
 COM_CONFIG_FIELD_DEFAULT_ACCESS_LEVEL_LABEL="Standard Zugriffsebene"
+COM_CONFIG_FIELD_DEFAULT_CAPTCHA_DESC="Die Standard Captcha-Einstellung für die Site auswählen. Eventuell müssen weitere Daten für das ausgewählte Captcha-Plugin unter „Plugins“ eingegeben werden."
 COM_CONFIG_FIELD_DEFAULT_CAPTCHA_LABEL="Standard <a href=\"https://de.wikipedia.org/wiki/CAPTCHA\" target=\"blank\">Captcha</a>"
+COM_CONFIG_FIELD_DEFAULT_EDITOR_DESC="Den Standard-Texteditor auswählen. Registrierte Benutzer haben die Möglichkeit diese Einstellung in ihrem persönlichen Profil wieder zu überschreiben, wenn diese Option ebenfalls aktiviert wurde."
 COM_CONFIG_FIELD_DEFAULT_EDITOR_LABEL="Standard Editor"
+COM_CONFIG_FIELD_DEFAULT_FEED_LIMIT_DESC="Die gewünschte Anzahl Beiträge auswählen, die in den Feeds angezeigt werden sollen."
 COM_CONFIG_FIELD_DEFAULT_FEED_LIMIT_LABEL="Standard Feed-Länge"
 COM_CONFIG_FIELD_DEFAULT_LIST_LIMIT_LABEL="Standard Listenlänge"
+COM_CONFIG_FIELD_ERROR_REPORTING_DESC="Die Empfindlichkeit der Fehlerbenachrichtigung kann aus der Drop-down-Liste ausgewählt werden.<br /><br /><strong>Hinweis:</strong><br />Auf den Hilfeseiten finden sich weitere Informationen."
```

```suggestion
COM_CONFIG_FIELD_ERROR_REPORTING_DESC="Die Empfindlichkeit der Fehlerbenachrichtigung kann aus der Drop-down-Liste ausgewählt werden.<br /><br /><strong>Hinweis:</strong><br />Auf den Hilfeseiten finden sich weitere Informationen."
```
Empfindlichkeiten hmm.

Auf den Hilfeseiten? Sollten wir die dann nicht verlinken?

---

@zero-24 see ee8573c966480a844fe71e5dc3b251c699a64c4e